tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dpv-helpers/data-loaded-check.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Check if data has been loaded and log a message if not.
+ *
+ * @param {Array | Map} contentMap - the map of content
+ * @return {boolean} haveContent - whether data has been loaded
+ */
+
+export const dataLoaded = (contentMap) => {
```

Comment:
If you destructure the arguments, you can simplify the places where you're calling this.

I.e.
```suggestion
export const dataLoaded = ({contentMap}) => {
```

would mean you could do 

```js
dataLoaded(dataMapper)
```

instead of

```js
dataLoaded(dataMapper.contentMap)
```